webdriver-spec.html

```diff
@@ -661,7 +661,13 @@
  a 128 bits long URN that requires no central registration process. [[!RFC4122]].
  <dfn>Generating a UUID</dfn> means Creating a
  <a>UUID</a> From Truly Random or Pseudo-Random Numbers [[!RFC4122]],
- and serializing it as appropriate for the callee.
+ and serializing it by converting the bit sequence to a string
+ representation, following the process outlined in [[!RFC4122]]. This
```

Comment:
"`and converting it to the string representation [[!RFC4122]].`" rather than all of this text.